index.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const puppeteer = require('puppeteer');
+const express = require('express');
+
+const main = async (answers) => {
+    try {
+        const port = 5001;
+        const destPath = path.join(process.cwd(), answers.destinationPath);
+        const fullResumePath = `http://localhost:${port}/${answers.sourcePath}`;

```

Comment:
variable should be renamed to `fullSourcePath`. This won't just be used for resumes

Reply:
Updated